pkg/util/tar_util_test.go

```diff
@@ -129,3 +130,60 @@ func createTar(testdir string, writer io.Writer) error {
 	}
 	return nil
 }
+
+func Test_CreateTarballOfDirectory(t *testing.T) {
+	tests := []struct {
+		name    string
+		wantF   string
+		wantErr bool
+	}{
+		{
```

Comment:
We don't need a table-driven test here if there's only one item. I'd just remove everything above and including `t.Run`.

Reply:
Removed.